### PR TITLE
Facebookios 4.24.0

### PR DESCRIPTION
**Pull request by SotoiGhost**
* Updated Facebook for iOS to version 4.24.0
* Updated Facebook Audience Network for iOS to version 4.24.0

### CODE REVIEW

File: Facebook.AudienceNetwork.iOS/source/Facebook.AudienceNetwork/ApiDefinition.cs [hunk 1]
```diff
@@ -420,6 +435,96 @@ interface MediaViewDelegate
...
+		void DisengageVideoSeek ();
+
+		// - (void)seekVideoToTime:(CMTime)time;
+		[Export ("seekVideoToTime:")]
+		void SeekVideoToTime (CMTime time);
+
+		// - (nullable id)addPeriodicTimeObserverForInterval:(CMTime)interval queue:(dispatch_queue_t) queue usingBlock:(void (^)(CMTime time))block;
+		[return: NullAllowed]
+		[Export ("addPeriodicTimeObserverForInterval:queue:usingBlock:")]
+		NSObject AddPeriodicTimeObserverForInterval (CMTime interval, DispatchQueue queue, Action<CMTime> block);
```
Comment (mattleibow) [inline Facebook.AudienceNetwork.iOS/source/Facebook.AudienceNetwork/ApiDefinition.cs]: Do we want the "ForInterval" suffix

File: Facebook.AudienceNetwork.iOS/source/Facebook.AudienceNetwork/Facebook.AudienceNetwork.csproj [hunk 2]
```diff
@@ -16,9 +16,9 @@
   <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
     <DebugSymbols>true</DebugSymbols>
     <DebugType>full</DebugType>
-    <Optimize>false</Optimize>
+    <Optimize>true</Optimize>
```
Comment (mattleibow) [inline Facebook.AudienceNetwork.iOS/source/Facebook.AudienceNetwork/Facebook.AudienceNetwork.csproj]: Is this intentional? optimize for debug.

Comment (mattleibow) [inline Facebook.AudienceNetwork.iOS/source/Facebook.AudienceNetwork/Facebook.AudienceNetwork.csproj]: I see the commit message.

File: Facebook.AudienceNetwork.iOS/source/Facebook.AudienceNetwork/ApiDefinition.cs (hunk 1 above)
Comment (SotoiGhost) [inline Facebook.AudienceNetwork.iOS/source/Facebook.AudienceNetwork/ApiDefinition.cs]: This can be omitted

**Pull request merged**